### DISCUSSION TITLE
When creating react projects with vite,module.css conflicts with px2rem

### DISCUSSION THREAD

**Discussion by LLiu0716** [postcss/postcss#1690] [Q&A]
Through the error prompt,find line 530 of lazy-result.js,change the error function to a number,and solve it,but I don't know the function's function.l hope the offcial can be compatible with the handling without the function.

**Comment by ai** [comment]
Can you show the origin error text and your diff change?

**Reply by LLiu0716** [reply]
`[vite] Internal server error: node.getIterator is not a function`
`at LazyResult.visitTick (G:\Desktop\vite-project\node_modules\postcss\lib\lazy-result.js:531:33)`
The flight code was found according to the error prompt
`visit.iterator = node.getIterator()`
`console.log(node.getIterator) // null`
This is an error-reporting functing.Through printing,it was found that this function was not defined.
`visit.iterator = (node.getIterator && node.getIterator()) || 12`
After making such modifications locally,my code can run normally.

**Reply by ai** [reply]
Seems like you have a different problem.

You have multiple versions of PostCSS in the system. Check `npm ls`.

It causes that AST tree contain nodes from different PostCSS versions and they are incompatible.

**Reply by LLiu0716** [reply]
ok,It seems that it's a problem with the px2rem package.Thank you for your answer.
